subgraph/schema.graphql

```diff
@@ -38,6 +38,25 @@ type CollectionRoleRevoked @entity(immutable: true) {
   transactionHash: Bytes!
 }
 
+type NewMint @entity(immutable: true) {
```

Comment:
Why do we need a new entitty? Can't we add these properties to the existing token entitty? I think we are still using entities the wrong way?

Reply:
No, this is the `NewMint` event entity and essential to handling the event and querying it in the future like other events in the schema file. `NewTokenName`, `Approval`, `GrantCollectionRole` all of these are also event entities and they are all immutable.